zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -272,10 +272,15 @@ public String getServerContextPath() {
   }
 
   public String getKeyStorePath() {
-    return getRelativeDir(
-            String.format("%s/%s",
-                    getConfDir(),
-                    getString(ConfVars.ZEPPELIN_SSL_KEYSTORE_PATH)));
+    String path = getString(ConfVars.ZEPPELIN_SSL_KEYSTORE_PATH);
+    if (path != null && path.startsWith("/")) {
```

Comment:
hmm, if `path` == `C:\zeppelin\conf\ssh`
isn't this turning it into a broken path like `c:\zeppelin\conf/c:\zeppelin\conf\ssh`?
```
getRelativeDir(
           String.format("%s/%s",
               getConfDir(),
               getString(path)));
```